README.md

```diff
@@ -0,0 +1,110 @@
+# Scalar Admin k8s
+
+## Usage of the CLI tool
+
+```console
+Usage: scalar-admin-k8s-cli [-h] [-d=<pauseDuration>] [-n=<namespace>]
+                            -r=<helmReleaseName> [-w=<maxPauseWaitTime>]
+                            [-z=<zoneId>]
+Scalar Admin pause tool for the Kubernetes environment
+  -d, --pause-duration=<pauseDuration>
+                             The duration of the pause period by millisecond.
+                               5000 (5 seconds) by default.
+  -h, --help                 Display the help message.
+  -n, --namespace=<namespace>
+                             Namespace that Scalar products you want to pause
+                               are deployed. `default` by default.
+  -r, --release-name=<helmReleaseName>
+                             Helm's release name that you specify when you run
+                               the `helm install <RELEASE_NAME>` command. You
+                               can see the <RELEASE_NAME> by using the `helm
+                               list` command.
+  -w, --max-pause-wait-time=<maxPauseWaitTime>
+                             The max wait time (in milliseconds) until Scalar
+                               products drain outstanding requests before they
+                               pause. If omitting this option, the max wait
+                               time will be the default value defined in the
+                               products. Most Scalar products have the default
+                               value of 30 seconds.
+  -z, --time-zone=<zoneId>   Specify a time zone ID, e.g., Asia/Tokyo, to
+                               output successful paused period. Note the time
+                               zone ID is case sensitive. Etc/UTC by default.
+```
+
+## Run the CLI tool on a Kubernetes environment
```

Comment:
This section explains how to run the `scalar-admin-k8s` CLI tool manually.
However, I think there are almost no users to use this way.
We assume that users use Helm Chart instead of that.